README.md

```diff
@@ -271,6 +271,34 @@ will return a presigned `uploadUrl`, which the client can use to upload the file
 The same goes for retrieving files. Caluma will respond with a presigned `downloadUrl` for
 the client to directly download the file from the storage provider.
 
+## Custom data sources
+For Choice- and MultipleChoiceQuestions it's sometimes necessary to populate the choices
+with calculated data or data from external sources.
+
+For this you can use the data_source extension point.
+
+An example data_source looks like this:
+
+ ```python
+from caluma.core.data_sources import BaseDataSource
```

Comment:
Isn't the import `caluma.data_source.data_sources`?

Reply:
Nope, ` caluma.core.data_sources.BaseDataSource` is the BaseClass you have to subclass in `caluma.data_source.data_sources`.

Reply:
@Yelinz already started on the frontend integration for this :wink:

Reply:
> @Yelinz already started on the frontend integration for this

Great! It's fixed now.